packages/flutter/lib/src/foundation/platform.dart

```diff
@@ -26,6 +26,12 @@ import '_platform_io.dart'
 ///
 /// Tests can also create specific platform tests by and adding a `variant:`
 /// argument to the test and using a [TargetPlatformVariant].
+///
+/// See also:
+///
+/// * [kIsWeb], a boolean which is true if the application is running on the
+///   web, where [defaultTargetPlatform] tells you what platform the browser is
```

Comment:
our style guide suggest avoid using `you`

Reply:
You are right. :-)  Reworded.